FizzBuzz 1/FizzBuzz/Program.cs

```diff
@@ -1,42 +1,117 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace FizzBuzz
 {
     class Program
     {
+        static Boolean ReadYN(string message)
+        {
+            char answer = 'a';
+            var valid = new List<char>();
+            valid.Add('y');
+            valid.Add('n');
+            while (!valid.Contains(answer))
```

Comment:
Having the list makes this a bit harder to read I think, I would just compare it to each character as there are only two 🙂